model_analyzer/config/input/config_command_profile.py

```diff
@@ -36,62 +37,6 @@
 from model_analyzer.triton.server.server_config import TritonServerConfig
 
 from .config_command import ConfigCommand
-from .config_defaults import (
```

Comment:
It was getting annoying (and ugly) to keep adding more defaults. So, I switched this to import all the defaults